vaccine_feed_ingest/runners/us/carbon_health/normalize.py

```diff
@@ -0,0 +1,278 @@
+#!/usr/bin/env python
+
+import datetime
+import os
+import pathlib
+import re
+import sys
+from typing import Any, List, Optional
+
+import orjson
+import us
+from vaccine_feed_ingest_schema import location as schema
+from vaccine_feed_ingest_schema.common import BaseModel
+
+from vaccine_feed_ingest.utils.log import getLogger
+from vaccine_feed_ingest.utils.normalize import normalize_zip
+
+# Performance optimization: skip validation in our pydantic models.
+#
+# This is safe to do because the ingestion framework will do the same
+# validation after running us.
+optimized = True
+if optimized:
+    BaseModel.create = BaseModel.construct
+else:
+    BaseModel.create = BaseModel.__init__
```

Comment:
Hmm, it looks like my suggestion didn't quite work :-/

I can work on a better solution, but in the meantime I would like to get this merged, and I would like to get it merged **without** the optimization at first (safety over speed), and increase the performance in a followup PR.

Are you okay with that plan?